src/euphorie/client/browser/templates/panel-add-user-to-organisation.pt

```diff
@@ -66,24 +66,47 @@
                     />
                     <tal:i18n i18n:translate="">Administrator</tal:i18n>
                   </label>
-                  <p class="pat-message info pat-depends"
-                     data-pat-depends="role=admin"
-                     i18n:translate=""
-                  >
+
+                  <tal:training_enabled tal:condition="view/is_training_enabled">
+                    <p class="pat-message info pat-depends"
+                       data-pat-depends="role=admin"
+                       i18n:translate=""
```

Comment:
I do prefer explicit msgids. That keeps already translated strings still valid even if a little text is changed in the original string.

Reply:
In general I agree. I judged it not worth the effort to change the existing strings in this case.

Reply:
Then again, I might as well do that now while there's not much else going on.

Reply:
Two things:

* i18ndude doesn't seem to support changing msgids. It marks the old translation as fuzzy and adds a new empty one. Fixing that manually for a dozen languages is tedious. Am I doing that wrong?
* It's good that I looked into it because it made me notice that the panel for editing organisation members uses the same texts and should be kept in sync.